.eslintrc.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   extends: '@mate-academy/eslint-config',
-  parser: "babel-eslint",
```

Comment:
The 'parser' property is missing. If you are using Babel, add 'parser: "babel-eslint"' to the configuration.

Reply:
>  If you are using Babel,
Am I?